webapp/src/ts/app.component.ts

```diff
@@ -277,10 +275,10 @@ export class AppComponent implements OnInit, AfterViewInit {
     // initialisation tasks that can occur after the UI has been rendered
     this.setupPromise = Promise.resolve()
       .then(() => this.chtScriptApiService.isInitialized())
-      .then(() => this.checkPrivacyPolicy())
       .then(() => this.initRulesEngine())
       .then(() => this.initTransitions())
       .then(() => this.initForms())
+      .then(() => this.checkPrivacyPolicy())
```

Comment:
I'm curious why this was moved?

Reply:
I actually don't remember why that was changed, it must have slipped by me. Doesn't seem like that would really do anything either, reverted to avoid unnecessary confusion.